### PR TITLE
Modest refactor & cEUR

### PR DESCRIPTION
**Pull request by tkporter**
* Changes imports from `github.com/ethereum/go-ethereum` -> `github.com/celo-org/celo-blockchain` following the transition made by celo-blockchain
* Uses `celotokens` from https://github.com/celo-org/kliento/pull/10 to monitor all celo tokens easily
* Modifies sorted oracles monitoring to look at rates for multiple stabletokens
* Renames the stability monitor to exchange, which is used for every stable token's exchange. Pulls registry-specific monitoring from the previous stability monitor into the registry monitor.
* Renames a bunch of prometheus metrics to be more intuitive, additionally adds labels to have some metrics play well with multiple tokens

TODO:
realizing now that the sorted oracles monitoring will need to work with non-stabletoken addresses, ie identifiers that are calculated like `address(keccak('CELOBTC'))`. I'll make this change as well

### CODE REVIEW

File: monitor/main.go [hunk 1]
```diff
@@ -176,7 +179,7 @@ func blockProcessor(ctx context.Context, startBlock *big.Int, headers <-chan *ty
 
 		parseAndLogEvent := func(logger log.Logger, eventIdx int, eventLog *types.Log) {
 			eventLogger := logger.New("logTxIndex", eventIdx, "logBlockIndex", eventLog.Index)
-			parsed, err := r.TryParseLog(transactionCtx, eventLog, h.Number)
+			parsed, err := r.TryParseLog(transactionCtx, *eventLog, h.Number)
```
Comment (yorhodes) [inline monitor/main.go:182]: why this change?
doesn't look like you changed the signature in the [kliento PR](https://github.com/celo-org/kliento/pull/10)

**Review by yorhodes — APPROVED** [review]
a bit hard to review with all the reordering but LGTM

File: monitor/main.go (hunk 1 above)
Comment (tkporter) [reply monitor/main.go:182]: Looks like there was a change to Kliento that Eksportisto had not yet adopted that changed the eventLog from a pointer to a value:
https://github.com/celo-org/kliento/commit/feecb3948ea292a2d81d2e668c092f195efca810#diff-5593a38c8a92809ab9448ee33f120e244e7fc4c0fd0689e239a9e85dc4ae0933L44

File: monitor/exchange.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+func (p exchangeProcessor) ObserveState(opts *bind.CallOpts) error {
+	// Exchange.ReserveFraction
+	reserveFraction, err := p.exchange.ReserveFraction(opts)
+	if err != nil {
+		return err
+	}
+
+	// TODO: This is a fraction and not actually an uint
```
[the remark this answers is not in the document]
Comment (tkporter) [reply monitor/exchange.go:59]: sorry what should I change? Should a float actually be returned by `p.exchange.ReserveFraction` instead of a big.Int?
